### DISCUSSION TITLE
Welcome to plugin.video.skyott Discussions!

### DISCUSSION THREAD

**Discussion by Paco8** [Paco8/plugin.video.skyott#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by tyshiro** [comment]
How do you install this?

**Reply by Paco8** [reply]
It's explained in the [README](https://github.com/Paco8/plugin.video.skyott/blob/main/README.md)

**Comment by tyshiro** [comment]
Hi, I was wondering if you how to install it to Kodi from this. I am very new to all this. I have kodi ob my Nvidia shield pro but I have no idea how to add things to it from the read me link you sent. I have always interest source manually. If you have a cheat note on how I could add the source, I would greatly appreciate it.Thank you in advance for your assistance,KrisSent from my iPhoneOn Dec 9, 2023, at 05:10, Paco8 ***@***.***> wrote:﻿
It's explained in the README

**Comment by Paco8** [comment]
Maybe this is the easiest way:

1. Visit [the release page](https://github.com/Paco8/plugin.video.skyott/releases) on GitHub.
2. Download the packages named plugin.video.skyott and script.module.ttml2ssa.
3. Copy these packages to the Download folder on your device.
4. Open Kodi and navigate to "Add-ons."
5. Choose "Install from zip."
6. Select the ttml2ssa package to install it.
7. Then, select and install the skyott package.

**Comment by tyshiro** [comment]
Thank you again and I’m so sorry to bothering you so much. I have the Nvidia shield pro TV. The problem is I don’t know how to copy things to the download folder on this device. Is there some kind of program I use to do this?Again, I am so sorry for driving you nuts on this. I tried googling the information, but nothing seems to come up on how to actually do it.Sent from my iPhoneOn Dec 9, 2023, at 09:52, Paco8 ***@***.***> wrote:﻿
Maybe this is the easiest way:

Visit the release page on GitHub.
Download the packages named plugin.video.skyott and script.module.ttml2ssa.
Copy these packages to the Download folder on your device.
Open Kodi and navigate to "Add-ons."
Choose "Install from zip."
Select the ttml2ssa package to install it.
Then, select and install the skyott package.

**Comment by Paco8** [comment]
You can copy the files connecting the nvidia shield using a usb cable or over the local network:
https://www.nvidia.com/en-us/shield/support/shield-tv-pro/how-to-transfer-content-to-shield/

There are also applications like this one:
https://play.google.com/store/apps/details?id=com.yablio.sendfilestotv

Or you can use an application like "downloader" to download the files directly in the nvidia shield:
https://play.google.com/store/apps/details?id=com.esaba.downloader

**Comment by tyshiro** [comment]
Thank you so much for being so helpful. I promise this will be the last question. I have downloader, but how do I transfer files into Kodi with downloader? I checked YouTube I checked online but they do not provide that answer on how to do it. can you help me with those instructions?Again, I am so sorry for bothering you. And thank you for being so patient and trying to teach me.Best regards,KrisSent from my iPhoneOn Dec 9, 2023, at 17:22, Paco8 ***@***.***> wrote:﻿
You can copy the files connecting the nvidia shield using a usb cable or over the local network:
https://www.nvidia.com/en-us/shield/support/shield-tv-pro/how-to-transfer-content-to-shield/
There are also applications like this one:
https://play.google.com/store/apps/details?id=com.yablio.sendfilestotv
Or you can use an application like "downloader" to download the files directly in the nvidia shield:
https://play.google.com/store/apps/details?id=com.esaba.downloader

**Reply by Paco8** [reply]
I think downloader saves the files in the folder Downloads/Downloader. So in Kodi go to add-ons, select install from zip, select External storage, Download, Downloader. The files should be there.

**Comment by tyshiro** [comment]
Thank you so much for taking the time to share all this with me. Best regards,KrisSent from my iPhoneOn Dec 10, 2023, at 09:01, Paco8 ***@***.***> wrote:﻿
I think downloader saves the files in the folder Downloads/Downloader. So in Kodi go to add-ons, select install from zip, select External storage, Download, Downloader. The files should be there.

**Comment by dadabear42** [comment]
Any chance or plan to release non-android version (Ubuntu/QNAP)?

Thanks a lot. I haven't recongized any discussion of incoming release for other platforms.

**Comment by zuzia-dev** [comment]
On Linux systems it cannot work due to restrictions on the SkyShowtime side.

**Comment by Paco8** [comment]
It doesn't work on Windows either because apparently they use something called verified media path (VMP), and only certified browsers are able to playback. That's why it doesn't work on Kodi.

But maybe I'm wrong, and there's a way to make playback work. If anyone knows a method, please let me know.

**Comment by w64** [comment]
Can you give us some recommendation, with which way for login in the addon, the session last longer? How often we have to login via the email & password and how often with the key?

**Comment by Paco8** [comment]
The session lasts the same with email or key file, about 6 months.

**Comment by gmjh123** [comment]
Sadly it only lasts 2 or 3 weeks for me. After 2 or 3 weeks my password is
not accepted and I have to generate the log file with a new password again.

**Comment by w64** [comment]
OK, i will test this. I'm resetting my password today (22/11/2024) and will write here, when the login stop working. Now, i choose to enter via the email and password. If i remember correctly, when login via the key (last time), the session last a few months.

**Comment by w64** [comment]
Activating with the email and password work only for 2 or 3 days. After that, Skyshowtime platform change your password and you must reset it, to login again. The new password cannot be same as the old one. I try to login with the key now. It shows error: unsupported browser (if the addon is used outside GoogleTV). Summary: using in other operating systems (if working), is not stable enough.

**Comment by Paco8** [comment]
Playback on PC doesn't work because the server that the addon uses to get the license is not working right now.

**Reply by w64** [reply]
OK, will you let us know when it works again? The people I was helping set up this addon have almost given up and are thinking about changing the streaming provider.

**Reply by Paco8** [reply]
I'm working on it.
